### PR TITLE
version 5 update

### PR DESCRIPTION
**Pull request by simondate**
Update to make work with v5

### CODE REVIEW

File: templates/game-frame.hbs [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-  <div class="component-widget gameIframe-widget">
+  <div class="component__widget gameIframe__widget">
     <div class="gameIframe-iframe-holder">
-      <img src="{{_loadingImg}}" />
-      <!-- You have the option of using a responsive iframe or a responsive object tag. PhaserJS games are not responsive
-      on iOS devices using the iFrame. Use the object tag instead.-->
-      <!-- <iframe scrolling="{{#if _scrolling}}{{_scrolling}}{{else}}no{{/if}}" class="gameIframe-iframe" src="">
-      </iframe> -->
-      <object type="text/html" data="" class="gameIframe-object"></object>
+      <iframe scrolling="{{#if _scrolling}}{{_scrolling}}{{else}}no{{/if}}" class="gameIframe-iframe" src="{{_source}}" style="height:{{_height}}px; width:{{_width}}px;">
```
Comment (anthkris) [inline templates/game-frame.hbs]: Hey @simondate What about this previous comment? Is the game responsive with this iframe?

**Review by anthkris — COMMENTED** [review]
Thank you so much for the PR, @simondate! I just had one question before approving.

File: templates/game-frame.hbs (hunk 1 above)
Comment (simondate) [reply templates/game-frame.hbs]: Hi @anthkris,

Sorry about leaving this in. I was just planning on creating a PR to get it to work with v5 and I must have left this line in.

I've just updated the commit to only include the changes to make it work with v5.

I've made some more changes on my fork too. Feel free to check them out and decide if you want them merged into here. 

As for iframe vs object — I'm not sure to be honest. You can make iframes responsive and also give them `width: 100%`. In the example I did I just set the game to always be a fixed width.

Comment (anthkris) [reply templates/game-frame.hbs]: Yeah, I know you can make iframes responsive, but there was an issue previously with them not being responsive on iOS. I haven't played with Adapt in a while, but I want to make sure that this still does what it's meant to.

Comment (anthkris) [reply templates/game-frame.hbs]: LGTM. I'll merge.

**Pull request merged**